plugins/test/fu-plugin-test.c

```diff
@@ -45,6 +45,13 @@ fu_plugin_destroy (FuPlugin *plugin)
 gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
+	if (g_getenv ("FWUPD_TESTS") == NULL) {
+                g_set_error (error,
+                             FWUPD_ERROR,
+                             FWUPD_ERROR_NOT_FOUND,
+                             "Test plugin is only used for continuous integration.");
```

Comment:
tabs/spaces were a casualty of copy paste, i'll fix that thanks for catching.
I'm unsure what you mean by full stop though, can you clarify?